frontend/src/pages/phone.page.tsx

```diff
@@ -33,7 +33,7 @@ const Phone: NextPage = () => {
 
   useEffect(() => {
     // check if phone flag is active - return to premium page if not.
-    if (!isFlagActive(runtimeData.data, "phones")) {
+    if (runtimeData.data && !isFlagActive(runtimeData.data, "phones")) {
```

Comment:
This prevents this check from failing before `runtimeData` is available.

Reply:
Ha, I had to add this check as well in https://github.com/mozilla/fx-private-relay/pull/2477/files#diff-b3d14fbc23006061636ccf3480d5f020d9b91671989a66ead4d6650c26c5bc81R47-R52 :)

Am I correct in thinking that this is a separate fix from the main thing in this PR (invalid phone number input)? If so, would you mind removing it from this one? Since it also gets fixed in my PR, which also moves a couple of other things around, so it would cause a conflict with that one.

Reply:
Removed